src/components/DataTable.js

```diff
@@ -179,7 +179,10 @@ export default ajaxCaller(class DataTable extends Component {
                       }
                     }
                   }, _.filter('visible', columnSettings))
-                ]
+                ],
+                styleCell: ({ rowIndex }) => {
+                  return rowIndex % 2 && { backgroundColor: colors.light(0.2) }
```

Comment:
It's not exactly but it's _really_ close to the background color of the header row. Can we adjust one or the other to create a little more contrast?

Reply:
I darkened the header slightly, how does that look?

Reply:
Ever so slightly better, I guess. I know there isn't much room to work with here. FWIW, I like it better on the MacBook screen than on my external monitor.

TOL: I suspect all of this subtlety may be lost when projected.